### ISSUE TITLE
attributes violation incorrectly triggered when using @Sendable

### ISSUE THREAD

**Issue by jose-pay** [realm/SwiftLint#3945]
### New Issue Checklist

- [x] Updated SwiftLint to the latest version
- [x] I searched for [existing GitHub issues](https://github.com/realm/SwiftLint/issues)

### Describe the bug

Attributes Violation: Attributes should be on their own lines in functions and types, but on the same line as variables and imports. (attributes)

The above rule is incorrectly triggered by the following code:
```swift
// This triggers a violation:
    @available(iOS, introduced: 14, obsoleted: 15)
    func refreshable(action: @escaping @Sendable () async -> Void) -> some View {
        modifier(RefreshableModifier(action: action))
    }
```

The warning goes away if `@Sendable` is removed.
